packages/shared/src/contexts/FeaturesContext.tsx

```diff
@@ -93,19 +98,20 @@ const getFeatures = (flags: IFlags): FeaturesData => {
 };
 
 export const FeaturesContextProvider = ({
+  isFeaturesLoaded,
   children,
   flags,
 }: FeaturesContextProviderProps): ReactElement => {
   const featuresFlags: FeaturesData = useMemo(() => {
-    const features = getFeatures(flags);
+    const features = getFeatures(flags, isFeaturesLoaded);
```

Comment:
I would recommend adding the `isFeaturesLoaded` prop after the `getFeatures` function.
I think we can keep the `getFeatures` implementation as it is. Makes more sense in my opinion